lms/urls.py

```diff
@@ -144,6 +144,7 @@
 
     url(r'^api/commerce/', include(('commerce.api.urls', 'lms.djangoapps.commerce'), namespace='commerce_api')),
     url(r'^api/credit/', include('openedx.core.djangoapps.credit.urls')),
+    url(r'^api/waffle_utils/', include('openedx.core.djangoapps.waffle_utils.urls')),
```

Comment:
Can we make this the final url that the plugin will have?  Should this be `api/toggles/`?